configure.py

```diff
@@ -319,6 +319,15 @@ def get_var(environ_cp,
     question += ' [y/N]: '
 
   var = environ_cp.get(var_name)
+  if var is not None:
+    _var_content = var.strip().lower()
+    if _var_content in ['1', 'y', 'true']:
```

Comment:
Do you want to also include 'yes' and 'no' as acceptable values?

Reply:
Indeed. Added "yes" and "no" to acceptable values. Thanks!